docs/command_line_reference.rst

```diff
@@ -674,11 +674,33 @@ Enable it with ``use_ssl:true``.
 **TLS/SSL Certificate Verification**
 
 Server certificate verification is controlled with the ``verify_certs`` boolean. The default value is `true`. To disable use ``verify_certs:false``.
-If ``verify_certs:true``, Rally will attempt to verify the certificate provided by Elasticsearch. If they are private certificates, you will also need to supply the private CA certificate using ``ca_certs:'/path/to/cacert.pem'``.
+If ``verify_certs:true``, Rally will attempt to verify the certificate provided by Elasticsearch. If they are private certificates, you will also need to supply the private CA certificate in PEM format using ``ca_certs:'/path/to/cacert.pem'``.
 
 You can also optionally present client certificates, e.g. if Elasticsearch has been configured with ``xpack.security.http.ssl.client_authentication: required`` (see also `Elasticsearch HTTP TLS/SSL settings <https://www.elastic.co/guide/en/elasticsearch/reference/current/security-settings.html#http-tls-ssl-settings>`_).
 Client certificates can be presented regardless of the ``verify_certs`` setting, but it's strongly recommended to always verify the server certificates.
 
+**CA & Client Certificate Format**
```

Comment:
I still have to test this, but can you please move the instructions after then "TLS/SSL Examples" section? Otherwise it isn't clear why we're talking about client_key and client_cert.

Reply:
> can you please move the instructions after then "TLS/SSL Examples" section?

👍🏼 Will do. I had actually considered the examples section initially, then thought it might be better to have client_cert and client_key before the examples.

Reply:
Ah, I see what you mean, but for better or worse, those examples are the only part where we currently define them. (Another option is to define them properly in the "Certificate Verification" section.)

That way, we can keep separate the **definition** of the option from their **conversion** from PKCS#12.

Reply:
Adjustments are in https://github.com/elastic/rally/pull/1403/commits/ee28cd159f7e5f5a42affea6c80cf251be51f5bd.